business_requirement/models/business.py

```diff
@@ -0,0 +1,446 @@
+        'res.users', string='Confirmed by',
+        copy=False,
+        readonly=True
+    )
+    responsible_id = fields.Many2one(
+        'res.users', string='Responsible',
+        copy=False,
+        readonly=True,
+        default=lambda self: self.env.user,
+        states={
+            'draft': [('readonly', False)],
+            'confirmed': [('readonly', False)]
+        }
+    )
+    reviewer_ids = fields.Many2many(
+        'res.users', string='Reviewers',
+        copy=False,
+        readonly=True,
+        states={
+            'draft': [('readonly', False)],
+            'confirmed': [('readonly', False)]
+        }
+    )
+    approval_date = fields.Datetime(
+        string='Approval Date',
+        copy=False,
+        readonly=True
+    )
+    approved_id = fields.Many2one(
+        'res.users',
+        string='Approved by',
+        copy=False,
+        readonly=True
+    )
+    company_id = fields.Many2one(
+        'res.company', string='Company',
+        required=True, readonly=True, states={'draft': [('readonly', False)]},
+        default=_get_default_company,
+    )
+    to_be_reviewed = fields.Boolean(
+        string='To be Reviewed'
+    )
+    kanban_state = fields.Selection(
+        [
+            ('normal', 'In Progress'),
+            ('on_hold', 'On Hold'),
+            ('done', 'Ready for next stage')
+        ],
+        'Kanban State',
+        track_visibility='onchange',
+        required=False,
+        default='normal'
+    )
+    origin = fields.Char(
+        string='Source',
+        readonly=True,
+        states={
+            'draft': [('readonly', False)],
+            'confirmed': [('readonly', True)]
+        }
+    )
+
+    @api.multi
+    @api.onchange('project_id')
+    def project_id_change(self):
+        if self.project_id and self.project_id.partner_id:
+            self.partner_id = self.project_id.partner_id.id
+
+    @api.model
+    def create(self, vals):
+        if vals.get('name', '/') == '/':
+            vals['name'] = self.env['ir.sequence'].next_by_code(
+                'business.requirement')
+        if vals.get('project_id'):
+            project_id = self.env['project.project']. \
+                browse(vals.get('project_id'))
+            if project_id and project_id.message_follower_ids:
+                msg_followers = []
+                for project in project_id.message_follower_ids:
+                    msg_vals = {
+                        'channel_id': project.channel_id.id,
+                        'display_name': project.display_name,
+                        'partner_id': project.partner_id.id,
+                        'res_model': self._name,
+                        'subtype_ids': project.subtype_ids.ids
+                    }
+                    msg_followers.append((0, 0, msg_vals))
+                if msg_followers:
+                    vals['message_follower_ids'] = msg_followers
+        return super(BusinessRequirement, self).create(vals)
```

Comment:
I think that you can take a look how to sale order add followers with sales team...

Reply:
project_id is deprecated and this part was related to